src/Repository/AdherentRepository.php

```diff
@@ -770,4 +771,25 @@ public function refresh(Adherent $adherent): void
     {
         $this->getEntityManager()->refresh($adherent);
     }
+
+    public function findPaginatedForMunicipalChief(
+        Adherent $municipalChief,
+        int $page = 1,
+        int $maxItemPerPage = 10
+    ): ApiPaginator {
+        if (!$municipalChief->isMunicipalChief()) {
+            return [];
```

Comment:
will not work with `: ApiPaginator` typehint.
what about renaming the method in:
`findPaginatedForInseeCodes(['insee1', 'insee2'], ...)` 
and remove this check:
`if (!$municipalChief->isMunicipalChief()) {`?